src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -640,23 +725,14 @@ $spacing: 4px;
 			border-left: 0!important;
 		}
 	}
-
-	&--button-variant.checkbox-radio-switch {
-		border: 2px solid var(--color-border-dark);
-		// better than setting border-radius on labels (producing a small gap)
-		overflow: hidden;
-
-		&__label {
-			text-overflow: ellipsis;
-		}
-
-		&--checked {
-			font-weight: bold;
-
-			label {
-				background-color: var(--color-primary-element-light);
-			}
-		}
+	&--button-variant-h-grouped &__label-text {
+		text-align: center;
+	}
+	&--button-variant-h-grouped &__label {
+		flex-direction: column;
+		justify-content: center;
+		width: 100%;
+		margin: 0;
```

Comment:
I would say to remove the gap for h-grouped buttons. Then the `NcAppSidebarTabs` would have the same height as :currently
```suggestion
		margin: 0;
		gap: 0;
```

Reply:
Ok